### ISSUE TITLE
Table never creates

### ISSUE THREAD

**Issue by kmickeletto** [lukasmartinelli/pgfutter#29]
Trying to import test data into PostgreSQL 9.1
`pgfutter --user postgres --port 30000 --schema public csv friends.csv`
`74 B / 74 B [====================] 100.00% 0`
`3 rows imported into public.friends`

No table is created though.  Looking in pg_log I see this...
`ERROR:  schema "public" already exists`
`STATEMENT:  CREATE SCHEMA public`

I have tried creating a new schema as well but still no joy.  This is on a CentOS 6.7 box.  Here is the version info if it helps.
`PostgreSQL 9.1.23 on x86_64-unknown-linux-gnu, compiled by gcc (GCC) 4.4.7 20120313 (Red Hat 4.4.7-17), 64-bit`

**Comment by lukasmartinelli** [comment]
> No table is created though. Looking in pg_log I see this...

It does try to create the schema but shouldn't be the cause for failing.
I must say I never tested it with PostgreSQL 9.1...

You are able to import it into the `import` schema?

**Comment by kmickeletto** [comment]
No, not into import schema either.  No matter if I use import or public, it
always says the schema already exists, even if it doesn't.  Do you have
plans to add support for it?  It really looks like the answer to my prayers
but I can't change the version of my postgres server.  Thanks so much for
the quick response!

Ken

**Comment by lukasmartinelli** [comment]
> No, not into import schema either.  No matter if I use import or public, it
> always says the schema already exists, even if it doesn't.

After PostgreSQL 9.3 there is support to only create schema if it does not exist yet.
But I don't think the schema creation is the problem here.

> Do you have
plans to add support for it?  It really looks like the answer to my prayers
but I can't change the version of my postgres server.  Thanks so much for
the quick response!

Can't promise. Perhaps I find a moment to make a quick test with it later and figure out what's wrong.

**Comment by kmickeletto** [comment]
Ok, I understand completely.  If I can provide any logs or test anything it
for you, please let me know.

I'm going to look at a different alternatives for the moment, but I am more
than happy to test out any new code and document findings.

Regardless of your ability to add support for 9.1, your tool looks amazing
and I appreciate you working to create such an awesome tool!

**Comment by danielmiq** [comment]
Also having this problem on windows 10. I'm connecting through VPN to my work server.
`
F:\>pgfutter_windows_amd64.exe --host 172.16.16.20 --port 5432 --username xxxxx --pass xxxx  --dbname gesermapa --schema PSR csv PSR2006_teste2.csv --d ";"
`
I get:
`pq: relation "PSR.psr2006_teste2" does not exist`

Version:

`PostgreSQL 9.3.9 on x86_64-unknown-linux-gnu, compiled by gcc (Debian 4.9.2-10) 4.9.2, 64-bit`


By the way, thank you for this tool.
